api/tests/tests.py

```diff
@@ -0,0 +1,75 @@
+import json
+from django.test import TestCase, Client
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from posts.models import Post, ContentType
+
+from .constants import POST_DATA, POST_IMG_DATA
```

Comment:
Might be less redundant if we import `POST_DATA` from `posts.tests.constants` instead